sklearn/metrics/ranking.py

```diff
@@ -197,18 +141,20 @@ def average_precision_score(y_true, y_score, average="macro",
     sample_weight : array-like of shape = [n_samples], optional
         Sample weights.
 
-    interpolation : string ['linear' (default), 'step']
-        Determines the kind of interpolation used when computed AUC. If there
-        are many repeated scores, 'step' is recommended to avoid under- or
-        over-estimating the AUC. See `Roam Analytics blog post
-        <https://github.com/roaminsight/roamresearch/blob/master/BlogPosts/
-        Average_precision/Average_precision_post.ipynb>`
-        for details.
-
-        ``'linear'``:
-            Linearly interpolates between operating points.
-        ``'step'``:
-            Uses a step function to interpolate between operating points.
+    interpolation : None (default), or string ['eleven_point']
+
+        ``None``:
```

Comment:
Seems to be okay.
<img width="761" alt="screen shot 2016-09-20 at 3 51 43 pm" src="https://cloud.githubusercontent.com/assets/7325675/18675412/3133f5bc-7f4a-11e6-9830-29039b0b1bc1.png">